domains/transfer/Transfer.API/API/Transfer/TransferAgreementProposalCleanup/TransferAgreementProposalCleanupService.cs

```diff
@@ -53,7 +53,7 @@ private async Task DeleteOldTransferAgreementProposals(DateTimeOffset olderThan,
         context.TransferAgreementProposals.RemoveRange(oldProposals);
 
         var activityLogEntries = oldProposals.Select(proposal => ActivityLogEntry.Create(Guid.Empty, ActivityLogEntry.ActorTypeEnum.System,
-            string.Empty, string.Empty, string.Empty, ActivityLogEntry.EntityTypeEnum.TransferAgreementProposal,
+            string.Empty, proposal.SenderCompanyTin, proposal.SenderCompanyName, ActivityLogEntry.EntityTypeEnum.TransferAgreementProposal,
```

Comment:
Why this change?

Reply:
This is a fix. Now we can actually see that the proposal has been deleted in the activity log. We could now before since the orgTin was string empty

Reply:
Yes, but how will the message look like then? Will it look like as if the senderCompanyName deleted the TA?

Reply:
Gonna check in vcluster. Asked Martin and he said I should put paramters like this

Reply:
Or actually this I can't test in vCluster since it takes 14 days to delete a proposal. I'm gonna trust Martin - the paramters are called organizationName and organizationTin so they must be related. We'll change it if it looks wierd. Ok with you?

Reply:
Make an integration test and check? See the output when you make a get request to activity log endpoint

Reply:
What I am asking is. If worker is deleting a proposal, should it then be logged in an activity log, as if it was organizationName and OrganizationTin that did it?

Reply:
If you see the code I have an integration test for this. I just can't see how the frontend interprets it